### DISCUSSION TITLE
2.5.3 更新日志

### DISCUSSION THREAD

**Discussion by LTCatt** [Meloong-Git/PCL#1618] [讨论]
· Java 下载与选择
　　在缺少 Java 时会自动下载所需的 Java（#99，@bwtx2023）
　　自动查找 Java 时将搜索新版官方启动器下载的 Java
　　手动导入 Java 后不再自动选择导入的 Java
· 资源搜索优化
　　添加 CurseForge / Modrinth 来源筛选（#1320，@TuanZiGit）
　　优化名称分段显示（#1567，@WForst-Breeze）
　　更新 Mod 中文名数据库
· 为下载管理和音乐播放按钮添加进度条（#580，@WForst-Breeze）
· 将 23w13a or b 分类为愚人节版本（#1545，@Chiloven945）
· 略微优化自动安装页面的性能（#1556，@anzheerdiya）
· 添加针对 COMException 的错误提示（#1564，by @youzi-2333）
· 支持 Mod 文件名包含特殊字符导致崩溃的分析（#1601，by @zkitefly）
· 更新 3 月赞助者列表，感谢大家的支持！
· 修复了以下 6 个 Bug
　　无法获取 MC 窗口导致报错（#1062，@150149）
　　无法下载部分 Fabric API（#1530，@snapshot6）
　　特定情况下刷新按钮会忽略输入（#1546，@GLMX22333）
　　错误地不允许使用 Java 19+ 启动 Fabric（#1574，@zhongbai2333）
　　自动进入服务器 设置在最新快照失效（#1598，@hejiehao）
　　Modrinth 前置 Mod 显示有误

<hr /><em>This discussion was created from the release <a href='https://github.com/Hex-Dragon/PCL2/releases/tag/2.5.3'>2.5.3</a>.</em>

**Comment by iPodtouch6557** [comment]
打扰一下，有个问题，2.5.3啥时候发？是github同步完就发吗？

**Reply by Chiloven945** [reply]
两三天之后发大概
